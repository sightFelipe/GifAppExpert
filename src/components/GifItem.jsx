const GifItem = ({title, url, id}) => {
    
  return (
    // <ol>
    //   <li key={img.id}> 
    //         {img.title}
    //   </li>
    // </ol>

    <div className="card">
      <img src={url} alt={title} />
      <p>{title}</p>
    </div>
  )
}

export default GifItem
