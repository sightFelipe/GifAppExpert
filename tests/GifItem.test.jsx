import { render } from "@testing-library/react"
import GifItem from "../src/components/GifItem"

describe('Tests in <GifItem/>', () => { 

    const title = 'Goku';
    const url = 'https://localhost/goku.jpg';	

    test('should match with snapshot', () => { 

           const {container} = render(<GifItem title='' url=''/>) 
           expect(container).toMatchSnapshot();
     }) 


 })