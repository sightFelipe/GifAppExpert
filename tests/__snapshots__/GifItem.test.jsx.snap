// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in <GifItem/> should match with snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt=""
      src=""
    />
    <p />
  </div>
</div>
`;
